universum/modules/automation_server/jenkins_server.py

```diff
@@ -29,13 +34,20 @@ def __init__(self, *args, **kwargs):
                                           "is not specified\n\n"
                                           "Please specify the url by using '--jenkins-trigger-url' ('-jtu')\n"
                                           "command-line option or URL environment variable.")
+        self.params = unify_argument_list(self.settings.param_list)
 
-    def trigger_build(self, revision):
-        processed_url = self.settings.trigger_url % revision
+    def trigger_build(self, param_dict=None):
+        processed_url = self.settings.trigger_url
+        if param_dict:
+            self.params.append(urllib.parse.urlencode(param_dict))
```

Comment:
`self.params` will grow larger on each call to `trigger_build`, which is counter intuitive.
Constructed object must keep it initial state.